CI Build failure introduced by commit "Use separate counters for SMS and phone calls."

https://travis-ci.org/github/healthchecks/healthchecks/builds/714508466
```
FAIL: test_it_sends_link (hc.accounts.tests.test_signup.SignupTestCase)
282----------------------------------------------------------------------
283Traceback (most recent call last):
284  File "/home/travis/build/healthchecks/healthchecks/hc/accounts/tests/test_signup.py", line 23, in test_it_sends_link
285    self.assertEqual(profile.sms_limit, 5)
286AssertionError: 500 != 5
```
Looks like it might be introduced by the following bug in the test, when `USE_PAYMENTS=False` is the default: 
```
        # A profile should have been created
        profile = Profile.objects.get()
        self.assertEqual(profile.sms_limit, 5)
        self.assertEqual(profile.call_limit, 0)
```
Ref: https://github.com/healthchecks/healthchecks/compare/77ee8452c5f8...8c134570371c#diff-9336213c71d0ba69ae89055c09e2c53aR21-R25

--- a/hc/accounts/tests/test_signup.py
+++ b/hc/accounts/tests/test_signup.py
@@ -20,8 +20,8 @@ class SignupTestCase(TestCase):
 
         # A profile should have been created
         profile = Profile.objects.get()
-        self.assertEqual(profile.sms_limit, 5)
-        self.assertEqual(profile.call_limit, 0)
+        self.assertEqual(profile.sms_limit, 500)
+        self.assertEqual(profile.call_limit, 500)
 
         # And email sent
         self.assertEqual(len(mail.outbox), 1)
@@ -42,7 +42,7 @@ class SignupTestCase(TestCase):
         channel = Channel.objects.get()
         self.assertEqual(channel.project, project)
 
-    @override_settings(USE_PAYMENTS=False)
+    @override_settings(USE_PAYMENTS=True)
     def test_it_sets_high_limits(self):
         form = {"identity": "alice@example.org"}
 
@@ -50,8 +50,8 @@ class SignupTestCase(TestCase):
 
         # A profile should have been created
         profile = Profile.objects.get()
-        self.assertEqual(profile.sms_limit, 500)
-        self.assertEqual(profile.call_limit, 500)
+        self.assertEqual(profile.sms_limit, 5)
+        self.assertEqual(profile.call_limit, 0)
 
     @override_settings(REGISTRATION_OPEN=False)
     def test_it_obeys_registration_open(self):

